File: two-sum/twoSum.py
def two_sum_bruteforce(nums, target):
    n = len(nums)
    for i in range(n):
        for j in range(i + 1, n):
            if nums[i] + nums[j] == target:
                return [i, j]  # Return the indices of the two numbers
    return []  # Return empty if no solution is found

# Example Usage:
nums = [2, 7, 11, 15]
target = 9
print(two_sum_bruteforce(nums, target))  # Output: [0, 1]

File: two-sum/twoSum.js
// Problem Statement
// Given an array of integers nums and an integer target, return the indices of the two numbers that add up to target.

// Using Brute Force (O(n²) Time Complexity)
function twoSum(nums, target) {
    for (let i = 0; i < nums.length; i++) {
        for (let j = i + 1; j < nums.length; j++) {
            if (nums[i] + nums[j] === target) {
                return [i, j];
            }
        }
    }
    return [];
}

// Example Usage:
console.log(twoSum([2, 7, 11, 15], 9)); // Output: [0, 1]
